src/index.ts

```diff
@@ -46,6 +51,7 @@ export default class HorizontalScroll extends EventEmitter {
         container = document.documentElement,
         showScrollbars = false,
         preventVerticalScroll = false,
+        ignoreScroll = IGNORE_SCROLL
```

Comment:
Remove the default ignore_scroll behaviour, as this is used for specific use cases.